Src/MoneyFox.Ui/Views/Statistics/CategorySpreading/StatisticCategorySpreadingPage.xaml

```diff
@@ -36,15 +36,47 @@
                         </ImageButton.Source>
                     </ImageButton>
                 </Grid>
-
+                
                 <Picker Grid.Row="1"
                         Title="{x:Static resources:Translations.SelectedPaymentTypeHeader}"
                         ItemsSource="{Binding PaymentTypes}"
                         SelectedItem="{Binding SelectedPaymentType, Mode=TwoWay}"
                         ItemDisplayBinding="{Binding ., Converter={StaticResource PaymentTypeStringConverter}}" />
 
+                <Grid Grid.Row="2" ColumnDefinitions="*, Auto" Padding="0,15,0,0" VerticalOptions="Start">
+                    <Label Style="{StaticResource TextBodySecondary}"
+                           Text="Number of categories to show:"
+                           VerticalOptions="Center"/>
+
+                    <FlexLayout Grid.Column="1"
+                                Direction="Row"
+                                JustifyContent="SpaceBetween">
+                        <Button Text="-"
```

Comment:
Couldn't you just use a Stepper here? I think that would simplify the code in the viewmodel as well.

Reply:
I could, I was actually thinking about using Stepper, but I wanted to have the Entry between the two buttons. Is it possible with Stepper?

Reply:
Mh, I don't think so. But then I would create your own little steper component and use that. Then you don't have to have Increment and Decrement commands in the  viewmodel of that concrete page, but can hide those in the component and just make a ValueChange Event.

Reply:
You're right. So I'll create a custom component. I'll let you know when it's ready.

Reply:
Hello, so I created a custom component called CustomStepperWithEntry and moved a lot of code from the viewmodel to its codebehind.